app/controllers/admin/nooks_controller.rb

```diff
@@ -100,5 +104,15 @@ def check_availability
       end
       render :js, template: 'admin/nooks/availability'
     end
+
+    def remove_photo
+      remain_photos = requested_resource.photos # copy the array
```

Comment:
I didn't know about the convention being followed. I'll update the code to reflect that appropriately.